ui-lib.pl

```diff
@@ -2280,6 +2280,31 @@ sub ui_radio_selector_javascript
 EOF
 }
 
+=head2 ui_switch_theme_javascript()
+
+The subroutine is designed to load JavaScript
+for switching themes using hotkeys.
+
+Hotkeys are:
+	To activate theme switch mode:
+	Use: `Ctrl+Alt+T` (or `Control+Option+T` on Mac).
+
+	Immediately after, within 1 second, select your desired theme by pressing:
+	- `Shift + A`: Authentic theme
+	- `Shift + G`: Gray theme
+	- `Shift + L`: Legacy theme.
+
+=cut
+
+sub ui_switch_theme_javascript
+{
+return &theme_ui_switch_theme_javascript(@_) if (defined(&theme_ui_switch_theme_javascript));
+return "" if (!$gconfig{'error_stack'} && !$gconfig{'debug_enabled'} && !$gconfig{'debug_theme_switcher'});
```

Comment:
Let's not enable this just because `error_stack` is turned on ... it should have a separate setting. A lot of users enable error_stack.

Reply:
Alright, I removed enabling on `error_stack` option.